.github/workflows/pathogen-repo-build.yaml

```diff
@@ -136,7 +136,12 @@ on:
           detaches from the build. Please use the `nextstrain build` command
           locally to reattach to AWS Batch builds to download outputs.
         type: string
-        default: ""
+        default: |
+          auspice/
+          results/
+          benchmarks/
+          logs/
+          .snakemake/log/
```

Comment:
What about defaulting these in the upload-artifact step instead of here in the parameter? That way the caller doesn't have to copy/paste these defaults just to add an additional file/dir to upload. The caller could still override a default path by providing an exclusion instead, e.g. `!logs/`.

Reply:
Ah, yeah. I can list the defaults in the description so that users can easily provide exclusions.

Reply:
Updated in https://github.com/nextstrain/.github/pull/44/commits/3f05452c6894e9b3f9ff50fb4c94c10650f5433f